### PR TITLE
Switch to a more standard Documenter setup

### LINKED ISSUE #348: Documentation is not deployed
Recent changes to the documentation have not been deployed, where “recent” = “since July 2019” (d52762d10dd0f46747bb6c50a5eb848f8111eaf6 has not been deployed). This means that the documentation pages linked in the README (both `stable` and `latest`) are quite outdated.

### PR DESCRIPTION
**Pull request by sostock**
Hopefully fixes #348. This does not use `mkdocs` anymore, so the theme is now the standard Documenter theme.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,18 +9,16 @@ julia:
...
-# uncomment the following lines to override the default test script
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'using Pkg; Pkg.build(); Pkg.test(coverage=true)'
-  - julia --compiled-modules=no -e 'using Pkg; Pkg.build(); Pkg.test(coverage=true)'
-after_success:
-  # push coverage results to Coveralls
-  - julia -e 'using Pkg; cd(Pkg.dir("Unitful")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
-  # push coverage results to Codecov
-  - julia -e 'using Pkg; cd(Pkg.dir("Unitful")); using Coverage; Codecov.submit(process_folder())'
```
Comment (giordano) [inline .travis.yml:24]: I guess we still want to run this, maybe with a single line:
```julia
after_success:
 - julia -e 'import Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder()); Codecov.submit(process_folder())'
```

Comment (giordano) [reply .travis.yml:24]: Oh, you moved up with the Travis keys `codecov` and `coveralls`

Comment (sostock) [reply .travis.yml:24]: Yeah, I used the Travis keys instead. It works for my packages.

**Comment by giordano** [comment]
Well, let's see if it works then

**Comment by giordano** [comment]
The documenter key is not setup correctly.  We need @ajkeller34 help here, I can change it in Travis but not the deploy key in the settings of this repository.

**Comment by ajkeller34** [comment]
Thanks for your patience. I think it is set up correctly now, but the directory must have changed at some point: http://painterqubits.github.io/Unitful.jl/dev/. Making a pull request momentarily.

**Pull request merged**